client/src/lib/stores/useRoomStore.ts

```diff
@@ -0,0 +1,113 @@
+import socketIoClient, { Socket } from 'socket.io-client'
+import { useEffect, useState } from 'react'
+import { useNavigate } from 'react-router-dom'
+import { v4 as uuidV4 } from 'uuid'
+import Peer from 'peerjs'
+import { ScreenNamesWithParams } from 'lib/routing'
+
+const WS = 'http://localhost:8080'
+
+const ws = socketIoClient(WS)
```

Comment:
shouldn't it be in the component body?

Reply:
i put this into body but doen't work mayebe i should add extra logic to handle this problem,  but i think this solution is acceptable. I also test demo video on heroku and ist work